config/config.go

```diff
@@ -87,7 +89,7 @@ var (
 		writer := writer.New(writer.WithLogger(opts.Logger), writer.WithDefaultLevel(log.ErrorLevel))
 
 		return []hclparse.Option{
-			hclparse.WithDiagnosticsWriter(writer, opts.DisableLogColors),
+			hclparse.WithDiagnosticsWriter(writer, true),
```

Comment:
Why is it disabled to `true`?

Reply:
Ah, changed for the debugging and forgot to revert. Thanks.